app/assets/src/styles/ui/controls/buttons/primary_button.scss

```diff
@@ -7,6 +7,7 @@
   color: $white;
 
   .icon-label {
+    position: relative;
```

Comment:
Was this a requirement for the label to position correctly?

Reply:
yes